numpyro/distributions/continuous.py

```diff
@@ -1720,20 +1728,20 @@ def __init__(
     def sample(self, key, sample_shape=()):
         assert is_prng_key(key)
 
-        shape = sample_shape + self.batch_shape[:-len(self.mv_normal.batch_shape)]
-        normal_samples = self.mv_normal.sample(sample_shape=shape, key=key)
+        shape = sample_shape + self.batch_shape[: -len(self.base_dist.batch_shape)]
+        normal_samples = self.base_dist.sample(sample_shape=shape, key=key)
```

Comment:
I think you can do `self.base_dist.expand(sample_shape + self.batch_shape).sample(key)` here